podcasts/EpisodeCell.swift

```diff
@@ -358,12 +358,14 @@ class EpisodeCell: ThemeableSwipeCell, MainEpisodeActionViewDelegate {
         updateCell(episodeUuid: episode.uuid)
     }
 
-    @objc private func updateCellFromSpecificEvent(_ notification: Notification) {
-        guard let episodeUuid = notification.object as? String, episodeUuid == episode?.uuid else {
-            return
-        }
+    @objc private func updateCellFromSpecificEvent(_ notification: Notification) {        
+        DispatchQueue.main.async { [weak self] in
```

Comment:
I'm unsure if this is the best solution, because with this change the finding of an episode on the `updateCell` method  will be done on the main thread.

Open to suggestion on alternatives